layouts/shortcodes/djm-install-troubleshooting.en.md

```diff
@@ -0,0 +1,10 @@
+The init script installs the Datadog Agent. To make sure it is properly installed, run the Agent status command:
```

Comment:
Let's start this as more of a FAQ section. e.g. let's have this content under the question:

I installed Data Jobs Monitoring but don't see any data in the product.
1. The init script installs...

Reply:
I see, thanks. For now, I believe "I installed Data Jobs Monitoring but don't see any data in the product." is the only question that this troubleshooting section aims to answer. I agree that checking the Agent status is only the first part of the answer, we intended to add a second point on checking the Tracer injection later.

I would be in favor of keeping the current structure for now so that the customer that we know is trying to set up DJM can find some answers quickly, and then plan in the backlog a task to create a more exhaustive FAQ. Is that ok for you ?

I've added a sentence at the top of the section, saying it specifically adresses the lack of data after installation in DJM.